assets/optional/cert-manager/release-cert-manager-aarch64.json

```diff
@@ -3,11 +3,11 @@
     "base": "4.20"
```

Comment:
BTW. this field we usually use to denote the version of the operand (in this case cert-manager), so it should be 1.17.z